lib/account-tasks/helpers/activateAndVoteHelper.ts

```diff
@@ -48,3 +48,20 @@ export async function activateAndVote(hre: HardhatRuntimeEnvironment, signer: Si
     }
   }
 }
+
+export async function getDefaultGroupsSafe(
```

Comment:
what is meant here by "safe"?

Reply:
It means that it cannot overflow, I will update the name i next PR so I dont have too many conflicts :-)